components/atom/tag/src/index.js

```diff
@@ -41,14 +41,15 @@ const filterKeys = (obj, listOfProps) =>
   }, {})
 
 const AtomTag = props => {
-  const {href, icon, onClick, size, responsive, type} = props
+  const {href, icon, onClick, size, responsive, type, extendedLabel} = props
   const isActionable = onClick || href
   const classNames = cx(
     'sui-AtomTag',
     `sui-AtomTag-${size}`,
     type && `sui-AtomTag--${type}`,
     responsive && 'sui-AtomTag--responsive',
-    icon && 'sui-AtomTag-hasIcon'
+    icon && 'sui-AtomTag-hasIcon',
+    extendedLabel && 'sui-AtomTag--extended-label'
```

Comment:
My opinion is that `extendedLabel`naming is not enough declarative. maybe a naming which contains the ellipsis content declaration should be more explicit